### PR TITLE
fix(core): only apply the animation/transition to the switch when the…

### PR DESCRIPTION
**Pull request by mikerodonnell89**
… switch is in the process of being toggled

part of #10171 

Adds a class to the fd-switch that removes any animation/transition. Removes this class while the checking/unchecking process is happening, allowing for the transition. This removes the flickering effect when the switch component is first rendered

### CODE REVIEW

File: libs/core/src/lib/switch/switch.component.scss [hunk 1]
```diff
@@ -7,3 +7,11 @@
 .fd-switch__invisible-text {
     display: none !important;
 }
+
+// to prevent the transition when the switch is first rendered
+.fd-switch-no-animate * {
+    -webkit-transition: none !important;
+    -moz-transition: none !important;
+    -ms-transition: none !important;
+    -o-transition: none !important;
```
Comment (N1XUS) [inline libs/core/src/lib/switch/switch.component.scss:16]: I think the browsers that we support already can use `transition` property without vendor prefixes

**Review by fkolar — APPROVED** [review]
Looks good to me. I am not sure if vendor prefixes are dead

File: libs/core/src/lib/switch/switch.component.scss (hunk 1 above)
Comment (fkolar) [reply libs/core/src/lib/switch/switch.component.scss:16]: I am not sure if there is already 100% support across all the browsers, desktop and mobile

Comment (N1XUS) [reply libs/core/src/lib/switch/switch.component.scss:16]: https://caniuse.com/?search=transition

Comment (N1XUS) [reply libs/core/src/lib/switch/switch.component.scss:16]: @fkolar we are supporting evergreen browsers, and they perfectly support transition property without vendor prefixes

Comment (fkolar) [reply libs/core/src/lib/switch/switch.component.scss:16]: Cool. so we just vendor prefixes and we should be good to go right ?

**Pull request merged**